krita.py

```diff
@@ -80,13 +81,42 @@ def define_schema(cls):
             node_id="ETN_KritaOutput",
             display_name="Krita Output",
             category="krita",
-            inputs=[io.Image.Input("images")],
+            inputs=[
+                io.Image.Input("images"),
+                io.Int.Input("width", default=0, min=0, optional=True),
```

Comment:
Maybe it's more convenient to use only a toggle (resize on/off), since the width/height are already part of the image. Unless you see a scenario where they don't match?